### PR TITLE
[PURCHASE-1749] Add auction result size filter

### PR DESCRIPTION
**Pull request by sepans**
Fixes [PURCHASE-1749](https://artsyproduct.atlassian.net/browse/PURCHASE-1749)

### Screenshot: 

Desktop:

Screen Shot 2020-02-13 at 2 54 33 PM

Mobile:

Screen Shot 2020-02-13 at 2 59 54 PM

### CODE REVIEW

File: src/Apps/Artist/Routes/AuctionResults/Components/AuctionFilters/index.tsx [hunk 1]
```diff
@@ -1,6 +1,12 @@
 import React from "react"
 import { AuctionHouseFilter } from "./AuctionHouseFilter"
+import { SizeFilter } from "./SizeFilter"
 
 export const AuctionFilters: React.FC = () => {
-  return <AuctionHouseFilter />
+  return (
+    <>
+      <SizeFilter />
+      <AuctionHouseFilter />
```
Comment (damassi) [inline src/Apps/Artist/Routes/AuctionResults/Components/AuctionFilters/index.tsx]: 👌

Comment (sepans) [reply src/Apps/Artist/Routes/AuctionResults/Components/AuctionFilters/index.tsx]: glad I used `context`. Its so easy adding new filters  🙏

**Comment by ashkan18** [comment]
🥇 going to merge!

**Comment by artsyit** [comment]
:rocket: PR was released in v25.9.1 :rocket:

**Pull request merged**